platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -0,0 +1,150 @@
+      set (RTX_LIB_PATH "${CMSIS_5_DIR}/CMSIS/RTOS2/RTX/Library/ARM/RTX_V8MBN.lib")
+    endif()
+elseif(COMPILER STREQUAL "GNUARM")
+    set (S_SCATTER_FILE_NAME   "${PLATFORM_DIR}/common/gcc/tfm_common_s.ld")
+    set (BL2_SCATTER_FILE_NAME "${NRF_COMMON_DIR}/gcc/nordic_nrf_bl2.ld")
+    set (NS_SCATTER_FILE_NAME  "${NRF_COMMON_DIR}/gcc/nordic_nrf_ns.ld")
+    if (DEFINED CMSIS_5_DIR)
+      # not all project defines CMSIS_5_DIR, only the ones that use it.
+      set (RTX_LIB_PATH "${CMSIS_5_DIR}/CMSIS/RTOS2/RTX/Library/GCC/libRTX_V8MBN.a")
+    endif()
+else()
+    message(FATAL_ERROR "No startup file is available for compiler '${CMAKE_C_COMPILER_ID}'.")
+endif()
+
+if (BL2)
+  set (BL2_LINKER_CONFIG ${BL2_SCATTER_FILE_NAME})
+  if (${MCUBOOT_UPGRADE_STRATEGY} STREQUAL "RAM_LOADING")
+      message(FATAL_ERROR "ERROR: RAM_LOADING upgrade strategy is not supported on target '${TARGET_PLATFORM}'.")
+  endif()
+  #FixMe: MCUBOOT_SIGN_RSA_LEN can be removed when ROTPK won't be hard coded in platform/ext/common/template/tfm_rotpk.c
+  #       instead independently loaded from secure code as a blob.
+  if (${MCUBOOT_SIGNATURE_TYPE} STREQUAL "RSA-2048")
+      add_definitions(-DMCUBOOT_SIGN_RSA_LEN=2048)
+  endif()
+  if (${MCUBOOT_SIGNATURE_TYPE} STREQUAL "RSA-3072")
+      add_definitions(-DMCUBOOT_SIGN_RSA_LEN=3072)
+  endif()
+endif()
+
+embedded_include_directories(PATH "${NRF_COMMON_DIR}" ABSOLUTE)
+embedded_include_directories(PATH "${NRFX_DIR}" ABSOLUTE)
+embedded_include_directories(PATH "${NRFX_DIR}/drivers/include" ABSOLUTE)
+embedded_include_directories(PATH "${NRFX_DIR}/mdk" ABSOLUTE)
+embedded_include_directories(PATH "${PLATFORM_DIR}/../include" ABSOLUTE)
+embedded_include_directories(PATH "${NRF_COMMON_DIR}/retarget" ABSOLUTE)
+embedded_include_directories(PATH "${NRF_COMMON_DIR}/Native_Driver" ABSOLUTE)
+embedded_include_directories(PATH "${PLATFORM_DIR}/cmsis" ABSOLUTE)
+
+
+if (NOT DEFINED BUILD_UART_STDOUT)
+  message(FATAL_ERROR "Configuration variable BUILD_UART_STDOUT (true|false) is undefined!")
+elseif(BUILD_UART_STDOUT)
+  list(APPEND ALL_SRC_C "${PLATFORM_DIR}/common/uart_stdout.c")
+  embedded_include_directories(PATH "${PLATFORM_DIR}/common" ABSOLUTE)
+  set(BUILD_NATIVE_DRIVERS true)
+  set(BUILD_CMSIS_DRIVERS true)
+endif()
+
+if (NOT DEFINED BUILD_NATIVE_DRIVERS)
+  message(FATAL_ERROR "Configuration variable BUILD_NATIVE_DRIVERS (true|false) is undefined!")
+elseif(BUILD_NATIVE_DRIVERS)
+  list(APPEND ALL_SRC_C "${NRFX_DIR}/drivers/src/nrfx_uarte.c"
+                        "${NRFX_DIR}/drivers/src/nrfx_nvmc.c"
+                        "${NRF_COMMON_DIR}/nrfx_glue.c"
+                        #"${NRF_COMMON_DIR}/Native_Driver/spu.c"
+                        )
+endif()
+
+if (NOT DEFINED BUILD_TIME)
+  message(FATAL_ERROR "Configuration variable BUILD_TIME (true|false) is undefined!")
+elseif(BUILD_TIME)
+  embedded_include_directories(PATH "${NRFX_DIR}/hal" ABSOLUTE)
```

Comment:
Remove this, please. nrfx HAL files should be included via `<hal/nrf_*.h>`.

Reply:
Removed